go/parquet/internal/utils/unpack_bool_arm64.go

```diff
@@ -14,20 +14,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !noasm
```

Comment:
I don't know too much about these, https://stackoverflow.com/questions/68360688/whats-the-difference-between-gobuild-and-build-directives